package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "name": "@serverless-dns/trie",
-  "version": "0.0.5",
+  "version": "0.0.15",
```

Comment:
`serverless-dns` doesn't use npm versions, btw. it uses `git tag`s ([see](https://github.com/serverless-dns/trie/tags)) instead.